### PR TITLE
Kill NO_VELLUM feature flag

### PR DESCRIPTION
**Pull request by orangejenny**
##### SUMMARY
Checked all servers, and no real projects are using this. Noticed it while looking at permissions in app manager.

Because there was a `no_vellum` attribute stored on the form, it's possible that a project had this turned on, marked some forms, and then turned it off. I think it's reasonable to say that if the flag is off, you no longer need access to the feature. @dimagi/product let me know if you disagree.

##### FEATURE FLAG
Allow disabling Form Builder per form (for custom forms that Vellum breaks)

##### RISK ASSESSMENT / QA PLAN
Small code removal, no QA.

### CODE REVIEW

File: corehq/apps/app_manager/templates/app_manager/partials/menu/form_link.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
     </a>
 
     <a id="view_form_{{ module.unique_id }}_{{ form.unique_id }}_sidebar"
-       {% if form.no_vellum %}
+       {% if form.form_type == 'shadow_form' %}
```
Comment (millerdev) [inline corehq/apps/app_manager/templates/app_manager/partials/menu/form_link.html:14]: Is this a new feature: shadow forms do not get vellum?

**Review by millerdev — APPROVED** [review]
Nice cleanup!

File: corehq/apps/app_manager/templates/app_manager/partials/menu/form_link.html (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/app_manager/templates/app_manager/partials/menu/form_link.html:14]: It's a few years old. Shadow forms are a variant of advanced forms. They're tied to a "source" form, and they use that form's XML and case actions but can also add their own case actions. They were using the `no_vellum` attribute, so I'm just switching to instead check the form type directly, it shouldn't change their behavior.

Comment (millerdev) [reply corehq/apps/app_manager/templates/app_manager/partials/menu/form_link.html:14]: Maybe the `no_vellum` field should be kept for just this purpose? Form types that cannot be edited in vellum can set a class attribute `no_vellum = True` (might be nice to reverse that logic to `can_edit_in_vellum = False` and have the default be `True`).

Comment (millerdev) [reply corehq/apps/app_manager/templates/app_manager/partials/menu/form_link.html:14]: Is this a couch field? If so, it should be private (with a leading underscore) so it is not saved in Couch.

Comment (orangejenny) [reply corehq/apps/app_manager/templates/app_manager/partials/menu/form_link.html:14]: I'm not following this. `form_type` is a couch field, yes, it's a property of `FormBase`. What do you mean about it not being saved in couch? I'm not keen on keeping `no_vellum` as its own attribute if it's synonymous with `form_type == 'shadow_form'`

Comment (orangejenny) [reply corehq/apps/app_manager/templates/app_manager/partials/menu/form_link.html:14]: I'd be fine with making a `FormBase.can_edit_in_vellum` property for readability, I'd just like to get rid of `no_vellum` as a data field.

Comment (millerdev) [reply corehq/apps/app_manager/templates/app_manager/partials/menu/form_link.html:14]: A `can_edit_in_vellum` property sounds good to me.

Comment (orangejenny) [reply corehq/apps/app_manager/templates/app_manager/partials/menu/form_link.html]: Cool, updated in 45855d65819

Comment (millerdev) [reply corehq/apps/app_manager/templates/app_manager/partials/menu/form_link.html]: Thanks!

**Review by millerdev — APPROVED** [review]

**Pull request merged**